org/admin.py

```diff
@@ -162,6 +168,181 @@ def get_inline_instances(self, request, obj=None) -> List[InlineModelAdmin]:  #
             obj and super().get_inline_instances(request, obj) or []
         )
 
+    def changelist_view(
```

Comment:
Does this need a permission check like the next method down?

Reply:
No, this is just overriding an internal method, it's not directly triggered by an end user.